README.md

```diff
@@ -523,50 +558,71 @@ Directory in which to store backups.
 
 ##### `backupdirmode`
 
-Permissions applied to the backup directory. This parameter is passed directly
-to the `file` resource.
+Permissions applied to the backup directory. This parameter is passed directly to the `file` resource.
 
 ##### `backupdirowner`
 
-Owner for the backup directory. This parameter is passed directly to the `file`
-resource.
+Owner for the backup directory. This parameter is passed directly to the `file` resource.
 
 ##### `backupdirgroup`
 
-Group owner for the backup directory. This parameter is passed directly to the
-`file` resource.
+Group owner for the backup directory. This parameter is passed directly to the `file` resource.
 
 ##### `backupcompress`
 
-Whether backups should be compressed. Valid values are true, false. Defaults to true.
+Whether backups should be compressed.
+
+Valid values are true, false.
+
+Defaults to true.
 
 ##### `backuprotate`
 
-How many days to keep backups. Valid value is an integer. Defaults to '30'.
+How many days to keep backups.
+
+Valid value is an integer.
+
+Defaults to '30'.
 
 ##### `delete_before_dump`
 
-Whether to delete old .sql files before backing up. Setting to true deletes old files before backing up, while setting to false deletes them after backup. Valid values are true, false. Defaults to false.
+Whether to delete old .sql files before backing up. Setting to true deletes old files before backing up, while setting to false deletes them after backup.
+
+Valid values are true, false.
+
+Defaults to false.
 
 ##### `backupdatabases`
 
 Specifies an array of databases to back up.
 
 ##### `file_per_database`
 
-Whether a separate file be used per database. Valid values are true, false. Defaults to false.
+Whether a separate file be used per database.
+
+Valid values are true, false.
+
+Defaults to false.
```

Comment:
`false`.

Actually, I'm going to stop marking these. But anyway, the values true and false should have backticks, except within a code sample